runtime/src/bank/address_lookup_table.rs

```diff
@@ -25,6 +26,20 @@ impl AddressLoader for &Bank {
     fn load_addresses(
         self,
         address_table_lookups: &[MessageAddressTableLookup],
+    ) -> Result<LoadedAddresses, AddressLoaderError> {
+        self.load_addresses_from_ref(
+            address_table_lookups
+                .iter()
+                .map(MessageAddressTableLookupRef::from),
+        )
```

Comment:
It is advantageous to add another indirection before calling the function? It seems to me that we are doing more work to first create the references than to call the function.

Reply:
Change here will not make this any faster - and you're right, it may even be slightly slower (copying a reference and 2 slice ptrs => ~40 bytes).

Reply:
^ To clarify - we do this mapping so we can use a shared implementation. Not for a specific benefit to the current function.

Additionally, the plan is for this function to **not** be called in our processing pipeline.